libraries/wutdevoptab/devoptab_fsa.h

```diff
@@ -116,12 +116,13 @@ int __wut_fsa_rmdir(struct _reent *r, const char *name);
 int __wut_fsa_utimes(struct _reent *r, const char *filename, const struct timeval times[2]);
 
 // devoptab_fsa_utils.c
-char *__wut_fsa_fixpath(struct _reent *r, const char *path);
+char *__wut_fsa_fixpath(struct _reent *r, const char *path, bool canonicalizedPath);
```

Comment:
Is there any reason not to just always normalise the path here?

Reply:
I just did to to avoid unnecessary canonicalizing the path when it's not needed. If you want to, we can just do it everywhere?